src/protocols/BUILD.gn

```diff
@@ -21,6 +21,8 @@ static_library("protocols") {
     "echo/Echo.h",
     "echo/EchoClient.cpp",
     "echo/EchoServer.cpp",
+    "message_counter/MessageCounterManager.cpp",
+    "message_counter/MessageCounterManager.h",
```

Comment:
MCSP is defined under SecureChannel protocol, we already have secure_channel directory under src/protocols, need to move from src/message_counter to src/protocols